packages/bot-web-ui/src/components/load-modal/local.jsx

```diff
@@ -8,8 +8,9 @@ import WorkspaceControl from './workspace-control.jsx';
 
 const LocalComponent = ({ handleFileChange, is_mobile, loaded_local_file, setLoadedLocalFile }) => {
     const file_input_ref = React.useRef(null);
+    const [is_file_supported, setFileSupport] = React.useState(true);
```

Comment:
Nit, but in general try to name the state update function consistent with the boolean, i.e. `setIsFileSupported` so the next developer knows that we're updating a boolean.